launch/vehicle_interface.launch.xml

```diff
@@ -6,13 +6,13 @@
   <arg name="max_throttle" default="0.4"/>
   <arg name="max_brake" default="0.8"/>
 
-  <arg name="pacmod_param_path" default="$(find-pkg-share as)/config/pacmod_param.yaml"/>
+  <arg name="pacmod_param_path" default="$(find-pkg-share as)/config/pacmod.param.yaml"/>
 
   <!-- pacmod3 -->
   <include file="$(find-pkg-share lexus_description)/launch/pacmod3_with_kvaser-node-launch.py">
-    <arg name="driver_params_path" value="$(find-pkg-share vehicle_launch)/config/$(var vehicle_id)/driver_params.yaml" />
-    <arg name="reader_params_path" value="$(find-pkg-share vehicle_launch)/config/$(var vehicle_id)/reader_params.yaml" />
-    <arg name="writer_params_path" value="$(find-pkg-share vehicle_launch)/config/$(var vehicle_id)/writer_params.yaml" />
+    <arg name="driver_params_path" value="$(find-pkg-share vehicle_launch)/config/$(var vehicle_id)/driver.param.yaml" />
+    <arg name="reader_params_path" value="$(find-pkg-share vehicle_launch)/config/$(var vehicle_id)/reader.param.yaml" />
+    <arg name="writer_params_path" value="$(find-pkg-share vehicle_launch)/config/$(var vehicle_id)/writer.param.yaml" />
```

Comment:
@wep21 Just a confirmation, will you use `*.param.yaml` in `vehicle_launch`, right?
It's different from upstream's name, but I think it's okay because they're in our repository.

Reply:
@kenji-miyake Yes, I intend to unify param file name conventions in our repository except for vendor packages.